NEWS.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### Changed
+* The JS API now uses camelCase identifiers.
+
+### Deprecated
+* Using kebab-case keys for the JS API is deprecated. Please use camelCase.
```

Comment:
Options currently formatted in kebab-case (i.e., with hyphens) are deprecated in favor of their camelCase variants, per JavaScript naming standards (#580)